client/src/cmdhfemrtd.c

```diff
@@ -1329,10 +1334,22 @@ static int cmd_hf_emrtd_dump(const char *Cmd) {
     
     if (CLIParamStrToBuf(arg_get_str(ctx, 2), dob, 6, &slen) != 0 || slen == 0) {
         BAC = false;
-    } 
+    } else {
+        if (slen != 6) {
```

Comment:
maybe not only check len,  but also if  MM is <=12 and DD <=31  (generic naive test)
or have  a array with MM, maxdd and make a lookup..

Reply:
Will do.